tools/batchTestGenerator/internal/generateBatchValues.go

```diff
@@ -16,3 +16,11 @@ func generateBatchValues(testSet testSetInput) (string, error) {
 
 	return b.String(), nil
 }
+
+func generateBachValuesStringArray(testSet testSetInput) ([]string, error) {
+	output := make([]string, 0)
```

Comment:
```suggestion
	output := make([]string, 0, len(testSet))
```

Probably doesn't make a huge difference, but setting the required capacity when it is known is a good practice that can save allocations in some circumstances.